snmp/tests/test_check.py

```diff
@@ -1071,15 +1071,6 @@ def test_timeout(aggregator, caplog):
     check.check(instance)
 
     aggregator.assert_service_check("snmp.can_check", status=SnmpCheck.WARNING, at_least=1)
-    # Some metrics still arrived
-    aggregator.assert_metric('snmp.ifInDiscards', count=4)
-    aggregator.assert_metric('snmp.ifInErrors', count=4)
-    aggregator.assert_metric('snmp.ifOutDiscards', count=4)
-    aggregator.assert_metric('snmp.ifOutErrors', count=4)
-    aggregator.assert_metric('snmp.sysUpTimeInstance', count=1)
-
-    common.assert_common_metrics(aggregator)
-    aggregator.all_metrics_asserted()
 
```

Comment:
I removed all those lines because:

- The calls are batched and because we are using sets (that do not guarantee the order), some metrics can randomly be missing (because batched along with the failing timeout call)
- metrics are not what we are interested in. We care much more about the WARNING and the timeout